versioned_docs/version-2.x/magic-button/x-ray.mdx

```diff
@@ -119,6 +119,14 @@ const MyCustomType = ({ aMyCustomType }) => {
 }
 ```
 
+By default, the X-Ray view will be scoped to the **block**.
```

Comment:
maybe: 

```suggestion
By default, the X-Ray view will be scoped as a **block**.
```

Reply:
I amended the commit with this suggestion.